### PR TITLE
[DOC] Clean up speech tics

### PR DESCRIPTION
**Pull request by haxscramper**
Removed the "ditto" speech tics from some comments.

### CODE REVIEW

File: lib/pure/random.nim [hunk 1]
```diff
@@ -145,7 +145,7 @@ proc next*(r: var Rand): uint64 =
   runnableExamples("-r:off"):
     var r = initRand(2019)
     assert r.next() == 13223559681708962501'u64 # implementation defined
-    assert r.next() == 7229677234260823147'u64 # ditto
+    assert r.next() == 7229677234260823147'u64 # implementation defined
```
Comment (haxscramper) [inline lib/pure/random.nim:148]: Found a funny "implementation defined" annotation that is probably going to be a huge pain to define properly.

**Comment by haxscramper** [comment]
Remote the "ditto" speech tics from some comments.

File: lib/pure/random.nim (hunk 1 above)
Comment (saem) [reply lib/pure/random.nim:148]: LoL, well that's fun.

File: doc/manual.rst [hunk 2]
```diff
@@ -3034,7 +3034,7 @@ Example:
     echo "permission denied"
   of "go-for-a-walk":     echo "please yourself"
   elif line.len == 0:     echo "empty" # optional, must come after `of` branches
-  else:                   echo "unknown command" # ditto
+  else:                   echo "unknown command"
```
Comment (saem) [inline doc/manual.rst]:
```suggestion
  else:                   echo "unknown command" # optional, must come after `of` and `elif` branches
```

File: tests/ccgbugs/tsamename3.nim [hunk 3]
```diff
@@ -8,7 +8,7 @@ block: # bug #15526
       x2: int
     let f2 = Foo(x2: 2)
 
-block: # ditto
+block:
```
Comment (saem) [inline tests/ccgbugs/tsamename3.nim]:
```suggestion
block: # bug #15526
```

It's this perhaps what was meant?

**Review by saem — COMMENTED** [review]
I think the ditto remarks might need some replication, with adjustment, some comments.

File: nimdoc/testproject/expected/subdir/subdir_b/utils.html [hunk 4]
```diff
@@ -193,10 +193,10 @@ <h1 class="title">subdir/subdir_b/utils</h1>
 </ul>
 
   </div>
-  
+
```
Comment (saem) [inline nimdoc/testproject/expected/subdir/subdir_b/utils.html]:
```suggestion
  
```

File: nimdoc/testproject/expected/subdir/subdir_b/utils.html [hunk 5]
```diff
@@ -193,10 +193,10 @@ <h1 class="title">subdir/subdir_b/utils</h1>
...
   <div class="nine columns" id="content">
   <div id="tocRoot"></div>
-  
+
```
Comment (saem) [inline nimdoc/testproject/expected/subdir/subdir_b/utils.html]:
```suggestion
  
```

**Review by saem — CHANGES_REQUESTED** [review]
re-added some spaces because these stupid tests are sensitive to this crap.

we can figure out how to solve that some other day, hopefully the suggestions should fix it for now.

**Pull request merged**